dev/parm/config/gfs/yaml/defaults.yaml

```diff
@@ -18,7 +18,8 @@ base:
   FHMAX_GFS: 120
   FHMAX_HF_GFS: 0
   FCST_BREAKPOINTS: ""
-  DO_GSISOILDA: "NO"
+  DO_GSISOILDA: "YES"
```

Comment:
I don't think we should turn either the soil DA or the soil IAU on by default yet.